NetSDK/Services/Client/Classes/SelfRefreshingClient.cs

```diff
@@ -90,8 +92,9 @@ private void ReadConfig(ConfigurationOptions config)
             HttpReadTimeout = config.ReadTimeout ?? 15000;
             SdkVersion = "net-" + Version.SplitSdkVersion;
             SdkSpecVersion = "net-" + Version.SplitSpecVersion;
-            SdkMachineName = config.SdkMachineName;
-            SdkMachineIP = config.SdkMachineIP;
+            SdkMachineName = Environment.MachineName;
+            var ipAdresses = Dns.GetHostAddresses(Environment.MachineName);
+            SdkMachineIP = ipAdresses.Last().ToString();
```

Comment:
what interface is this picking up?

it wouldn't hurt leaving the config param for those people that want to specify the IP address through some automation or bash or equivalente in windows. thoughts?